docs/home.mdx

```diff
@@ -16,7 +16,7 @@ If you encounter any issues or have questions about EssentialsX, feel free to jo
 ## Reporting Bugs
 Encountered a bug? Help us improve EssentialsX by reporting it. Here’s how you can report a bug effectively:
 
-* Check Existing Reports: Before submitting a new bug report, please check our Bug Tracker to see if the issue has already been reported. This helps avoid duplicate reports.
+* Check Existing Reports: Before submitting a new bug report, please check our [Bug Tracker](https://github.com/EssentialsX/Essentials/labels/bug%3A%20confirmed) to see if the issue has already been reported. This helps avoid duplicate reports.
```

Comment:
This should not be using the "confirmed" label, only because there are reports we don't always get the chance to do triage on.

Reply:
This has been addressed by longer including both confirmed and unconfirmed labels for the Bug Trucker masked link.